### PR TITLE
Add travis (fix #141)

### LINKED ISSUE #141: make travis CI check pull requests
it should build rpm with the help of tito (depends #129) and report the success / failure.
[1] https://github.com/spacewalkproject/spacewalk/tree/master/rel-eng/travis

### PR DESCRIPTION
**Pull request by PavolVican**

### CODE REVIEW

**Comment by PavolVican** [comment]
Travis must be allowed to rh-lab-q/rpg

File: rel-eng/travis/build.sh [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+# our package RPG
+package="rpg"
+package_basename=$(basename $package)
+# process package
+echo -en "travis_fold:start:$package_basename-build\\r"
+echo "Building $package_basename"
+srpm=$(tito build --srpm --test | awk 'NF{p=$0}END{print p}' | sed 's/^Wrote: //g')
+echo "SRPM created: $srpm"
+# decide arch
+case $package_basename in
```
Comment (jsilhan) [inline rel-eng/travis/build.sh]: why you have this condition?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  - master
+notifications:
+  email: false
+sudo: required
+before_install:
+- echo "deb http://us.archive.ubuntu.com/ubuntu trusty main universe" | sudo tee -a /etc/apt/sources.list
+- sudo apt-get update
+- wget https://github.com/dgoodwin/tito/archive/tito-0.4.18-1.tar.gz -O /tmp/tito.tar.gz
+install:
+- sudo apt-get install yum yum-utils mock pigz python-rpm python-openssl python-urlgrabber python-decoratortools python-lzma python-setuptools python-krbv rpm
```
Comment (jsilhan) [inline .travis.yml]: Do we really need them all?

File: rpg.spec [hunk 3]
```diff
@@ -30,7 +30,7 @@ packaged files. Beginners can get familiar with packaging process or the
 advanced users can use our tool for a quick creation of a package.
 
 %prep
-%autosetup
+%setup -q -n rpg-%{version}
```
Comment (jsilhan) [inline rpg.spec]: add this to separate commit, please, as this fixes the rpg rpm building

**Comment by jsilhan** [comment]
> Travis must be allowed to rh-lab-q/rpg

So what should I enable and where?

**Comment by PavolVican** [comment]
Sign in with github account on the website https://travis-ci.org where you have this repository "rh-lab-q/rpg" and you enable in settings Build pushes, Build pull requests

**Comment by PavolVican** [comment]
All done!

**Comment by jsilhan** [comment]
It looks good. I just can't see organizations I am owning from Travis even though I set them to be public visible.

**Comment by jsilhan** [comment]
Now travis showed me the rh-lab-q/rpg repo. Merged, thanks/

**Pull request closed without merging**